ImmuDB4Net/KVMetadata.cs

```diff
@@ -123,16 +142,27 @@ public bool HasExpirationTime
         }
     }
 
-    public long ExpirationTime()
+    /// <summary>
+    /// Gets the expiration time in epoch microsecond
```

Comment:
This one actually uses seconds precision.

Maybe better to operate on some DateTime structure?

Reply:
changed to DateTime

Reply:
changed to DateTime